triage/labels.rst

```diff
@@ -10,405 +10,135 @@ Many labels are shared for both use cases, while some are dedicated
 
-:gh-label:`type-tests`
-    Used for issues/PRs that exclusively involve changes to the tests.
+* :gh-label:`stdlib`: for standard library modules in the :cpy-file:`Lib`
+  directory (written in Python).
+* :gh-label:`extension-modules`: for standard library modules in the
+  :cpy-file:`Modules` directory (written in C).
+* :gh-label:`interpreter-core`: for changes related to the interpreter core in
+  the :cpy-file:`Objects`, :cpy-file:`Python`, :cpy-file:`Grammar`,
+  and :cpy-file:`Parser` dirs (written mostly in C).
+* :gh-label:`docs`: for documentation in the :cpy-file:`Doc` directory
+  (written in reStructuredText), docstrings, and code comments.
+* :gh-label:`tests`: for tests in the :cpy-file:`Lib/test` directory
+  (written in Python) and changes other related to tests, ``unittest``,
+  or ``doctests``.
 
-:gh-label:`OS-Mac` / :gh-label:`OS-Windows`
-    Used for issues/PRs involving changes which only have an effect upon
-    a specific operating system.
 
-:gh-label:`spam`
-    Used for issues/PRs that don't include enough eggs or bacon.
+Expert labels
+=============
 
-Labels specific to issues
-=========================
+These labels are used to specify the area of expertise required to address
+the issue/PR.  This includes both specific modules/packages and generic
+interest areas.
 
-Priority
---------
+Adding these labels is also a way to notify the relevant experts, since
+they are encouraged to subscribe to these labels.  Depending on the label,
+this might also automatically add the issue to a GitHub project.
 
-:gh-label:`release-blocker`
-    The highest priority of an issue. If unaddressed, will cause the
-    release manager to hold releasing a new version of Python.
+You can see the `full list of expert labels on GitHub
+<https://github.com/python/cpython/labels?q=expert>`_.
 
-:gh-label:`deferred-blocker`
-    A release blocker that was pushed one or more releases into the
-    future. Possibly a temporary workaround was employed, or the version
-    of Python the issue is affecting is still in alpha or beta stages
-    of development.
 
-Component
----------
+OS labels
+=========
 
-:gh-label:`library`
-    Used for issues involving Python modules in the ``Lib/`` dir.
+These labels are used to specify which operating systems are affected.
+Since most issues either affect all systems or are specific to Unix,
+the only available labels are :gh-label:`OS-windows`, :gh-label:`OS-mac`,
+and :gh-label:`OS-freebsd`.
 
-:gh-label:`docs`
-    Used for issues involving documentation in the ``Doc/`` dir.
 
-:gh-label:`interpreter-core`
-    Used for issues in interpreter core (``Objects/``, ``Python/``,
-    ``Grammar/``, and ``Parser/`` dirs).
+Version labels
+==============
 
-:gh-label:`extension-modules`
-    Used for issues involving C modules in the ``Modules/`` dir.
+These labels are used to indicate which versions of Python are affected.
+The available version labels (with the form :samp:`3.{x}`) are updated
```

Comment:
I think 3.x is fine here and much more common. I can't recall seeing, or find with a quick search, any Python 3.y.

Things like PEP 440 or the SemVer spec are different, when discussing X.Y.Z and their relationship, but here the X is the first unknown. (And the PEP has much more N.N than X.Y.)

Reply:
In that case, then,

```suggestion
The available version labels (with the form :samp:`3.{N}`) are updated
```

Reply:
Just to clarify, the usage I've typically seen for `3.x` is _any 3.* version_, _all 3.* versions_, etc., whereas here the labels have a _specific_ `3.{N}` version, with the meaning of the character being `{}` (i.e. `.{N}`) rather than `*` (i.e. `.x`).